config.sh

```diff
@@ -115,3 +115,9 @@ OVNKUBE_STREAM=ovn-kubernetes
 OVNKUBE_GIT_URI=https://github.com/openshift-kni/ovn-kubernetes.git
 OVNKUBE_GIT_REF=4.3-ipv6
 OVNKUBE_DOCKERFILE=Dockerfile
+
+#aws-machine-controllers build config
+AMC_STREAM=aws-machine-controllers
+AMC_GIT_URI=https://github.com/openshift-kni/cluster-api-provider-aws
+AMC_GIT_REF=4.3-ipv6
+AMC_DOCKERFILE=Dockerfile
```

Comment:
when we create each image here, it's good to check in the openshift/release repo to determine which Dockerfile is used to build the image if there's more than one Dockerfile in the repo.  Just FYI

Reply:
`Dockerfile` is correct in this case, see https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-release-4.3.yaml#L13-L16

Reply:
Thanks found the linkage for the name and confirmed the Dockerfile name

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-release-4.3.yaml#L14